### DISCUSSION TITLE
You must install .NET runtime error even tho its installed

### DISCUSSION THREAD

**Discussion by fpsprohd** [OpenTabletDriver/OpenTabletDriver#3299] [General]
### Steps to Reproduce

open the app and get the error pop up

### Expected Behavior

open the app and be able to use open tablet driver properly

### Observed Behavior

i have installed .net SDK 7.0.408 , SDK 7.0.315 , SDK 7.0.118 and .NET Runtime 7.0.18 and .net runtime 5 based on an old github bug post

### Tablet Device

wacom-ctl-472

### Diagnostic Information

N\A

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
You need **desktop runtime**. https://opentabletdriver.net/Wiki/Install/Windows
